Add warning in case of opting-out for data forwarding (#3440)
uyuni-project/uyuni-docs: modules/administration/pages/task-schedules.adoc

----
+

[WARNING]
====

> Disabling data sync with SCC will lead to reduced visibility of your managed clients between RMT, SMT, SUSE Manager and SCC	directly registered clients.
> 
> By syncing data, you ensure a uniform view of all registered clients.
> 
> https://suselinux.fra1.qualtrics.com/jfe/form/SV_0ooNnrY0rYuQScS[Help us improve our services by sharing your reason for opting out]

Reviewer: ```suggestion
Disabling data synchronizing with SCC will lead to reduced visibility of your managed clients between RMT, SMT, {productname} and SCC-directly registered clients.

By synchronizing data, you ensure a uniform view of all registered clients.

For more information, see https://suselinux.fra1.qualtrics.com/jfe/form/SV_0ooNnrY0rYuQScS[Help us improve our services by sharing your reason for opting out].
```

> https://suselinux.fra1.qualtrics.com/jfe/form/SV_0ooNnrY0rYuQScS[Help us improve our services by sharing your reason for opting out].

Reviewer: ```suggestion
https://suselinux.fra1.qualtrics.com/jfe/form/SV_0ooNnrY0rYuQScS[Help us improve our services by sharing your reason for opting out]
```

> For more information, see https://suselinux.fra1.qualtrics.com/jfe/form/SV_0ooNnrY0rYuQScS[Help us improve our services by sharing your reason for opting out].

Author: ```suggestion
https://suselinux.fra1.qualtrics.com/jfe/form/SV_0ooNnrY0rYuQScS[Help us improve our services by sharing your reason for opting out].
```
